packages/types/src/srml/parachains/definitions.ts

```diff
@@ -0,0 +1,94 @@
+// Copyright 2017-2019 @polkadot/types authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+export default {
```

Comment:
This is where we actually define the parachain types in a JSON-like format - and it uses the JSON injection path.

Reply:
The alternative would just be that the `getTypeDef` does not throw (if set as not-throwing), allowing at least the generation to work. However, this may create problems around the injection, i.e. the above will solve the ordering imports for the type extraction, but may not for actual use creation - to be looked at.

Reply:
Wanted to log this explicitly, but not worth the effort when we are moving to reflected - depends on the timing, it will hit the contracts first.